DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/SectionContentViewModelTests.cs

```diff
@@ -56,24 +57,112 @@ public void Section_content_should_have_section_name()
         }
 
         [Test]
-        public void Section_content_without_has_learning_should_have_empty_percent_complete()
+        public void Section_content_should_have_consolidation_path_when_courseSetting_exercise_is_null()
         {
             // Given
-            const bool hasLearning = false;
+            const string consolidationPath = "consolidation/path";
+            var sectionContent = SectionContentHelper.CreateDefaultSectionContent(
+                consolidationPath: consolidationPath,
+                courseSettings: null
+            );
+            var expectedConsolidationPath = ContentUrlHelper.GetContentPath(config, consolidationPath);
+
+            // When
+            var sectionContentViewModel = new SectionContentViewModel(config, sectionContent, CustomisationId, SectionId);
+
+            // Then
+            sectionContentViewModel.ConsolidationExercisePath.Should().Be(expectedConsolidationPath);
+        }
+
+        [Test]
+        public void Section_content_should_have_setting_consolidation_path_when_courseSetting_exercise_is_not_null()
+        {
+            // Given
+            const string consolidationPath = "consolidation/path";
+            const string consolidationPathFromSettings = "consolidation/path/fromsetting";
```

Comment:
Ah no I don't think this is what this setting is for. It's meant to be the text you show on the link to the consolidation exercise. It should be "Consolidation exercise" by default, but it could be set to something else such as "Extra exercise" or something. Does that make sense? 🙂

Reply:
Ah I had misread the ticket, sorry about that! Fixed in a94e772